plugins/woocommerce/includes/class-wc-order-item.php

```diff
@@ -52,6 +52,14 @@ class WC_Order_Item extends WC_Data implements ArrayAccess {
 	 */
 	protected $object_type = 'order_item';
 
+	/**
+	 * Legacy package key.
+	 *
+	 * @deprecated 4.4.0 For legacy actions.
+	 * @var string
+	 */
+	public $legacy_package_key;
+
```

Comment:
Also this.

Reply:
https://github.com/woocommerce/woocommerce/pull/44181#discussion_r1474794763